examples/nextjs/package.json

```diff
@@ -10,10 +10,14 @@
   },
   "dependencies": {
     "next": "^12.1.4",
-    "wicg-inert": "^3.1.1"
+    "wicg-inert": "^3.1.1",
+    "nuka-carousel": "workspace:nuka-carousel@*"
```

Comment:
I think this should just be `"*"`? PNPM in workspace mode should auto-infer this already and we otherwise have to check this works correctly with `changesets version`

Reply:
I changed it to this then realized it was using the pnpm cache. I think I need to figure out how to link it to `packages\nuka\src\index.tsx` over `lib` or any other output so developers can iterate on Nuka and have the source changes show up in the working demo.

Reply:
Yep. So, reading up, I think this still _should_ work as we have it. But. here's an `.npmrc` option that forces the link in my check out of this repo with just `"*"` as the specifier:

```diff
diff --git a/.npmrc b/.npmrc
index 319e41e..a018756 100644
--- a/.npmrc
+++ b/.npmrc
@@ -1 +1,2 @@
 strict-peer-dependencies=false
+prefer-workspace-packages=true
```